### DISCUSSION TITLE
Inquiry Regarding DotMim.Sync (DMS) for bidirectional sync Project

### DISCUSSION THREAD

**Discussion by almuatacim** [Mimetis/Dotmim.Sync#1227] [Q&A] [answered]
Hello,

I'm exploring DotMim.Sync (DMS) for SQL Server database synchronization in a project with a central hub and multiple branches. I'm interested in bidirectional sync, selective table/column sync, conflict resolution, error handling, and secure data transmission. Can DMS handle bidirectional sync effectively? Does it allow column-level sync? How does it manage conflicts and errors, and what security features does it offer for sensitive data?

Thanks for your insights.

**Comment by Mimetis** [comment, marked as answer]
You have full documentation here : https://dotmimsync.readthedocs.io/

Regarding all your questions, I would say Yes, but it really depends on your scenario.
